README.md

```diff
@@ -63,6 +68,8 @@ Core Libraries are different from GHC boot libraries: some boot libraries are no
 (and as such are maintained either solely by GHC Team, or by third parties), and some
 Core Libraries are not required to boot GHC.
 
+Questions related to management of Core Libraries are usually discussed in respective issue trackers. Please bring them to CLC attention by CC'ing CLC members or pinging by email to core-libraries-committee at haskell dot org.
+
 ### Suggesting changes to Core libraries
 
 Because the CLC owns these libraries but does not maintain them, if you have an improvement
```

Comment:
I think it might help to say what a Core Library **is**!

Something like.  
```
The Core Libraries are libraries 
* That are depended on by many other packages, and hence play a particularly important role in the Haskell ecosystem.
* Whose maintainers have agreed (or requested) support from the Core Libraries Committee in ensuring the long-term health of the library.

The overall goals are
* to reduce pressure on, and and add support for, maintainers whose libraries have grown very popular; and 
* to provide the commuinty with a way to deal with the single-point-of-failure problem when a maintainer of a widely-used library becomes unavailable (perhaps for good reasons).

Reply:
Thanks, done.